config.json

```diff
@@ -1,21 +0,0 @@
-{
```

Comment:
It's already in .gitignore and you didn't fix senpai.js to rebuild it if it's not there.
Either do that or don't remove it.